### PR TITLE
added more details to the instruction

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7898

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ As part of the Endpoint Detection and Response capabilities, Microsoft Defender
 >JAMF doesn’t have built-in support for content filtering policies, which are a pre-requisite for enabling the network extensions that Microsoft Defender ATP for Mac installs on the device. Furthermore, JAMF sometimes changes the content of the policies being deployed.
 >As such, the following steps provide a workaround that involve signing the configuration profile.
 
-1. Save the following content to your device as `com.microsoft.network-extension.mobileconfig`
+1. Save the following content to your device as `com.microsoft.network-extension.mobileconfig` using a text editor
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md:62]:
```suggestion
1. Save the following content to your device as `com.microsoft.network-extension.mobileconfig` using a text editor:
```

File: windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md [hunk 2]
```diff
@@ -122,21 +122,38 @@ As part of the Endpoint Detection and Response capabilities, Microsoft Defender
     </plist>
     ```
 
-2. Verify that the above file was copied correctly. From the Terminal, run the following command and verify that it outputs `OK`:
+2. Verify that the above file was copied correctly by running `plutil` utility in the Terminal:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md:125]:
```suggestion
2. Verify that the above file was copied correctly by running the `plutil` utility in the Terminal:
```

File: windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md [hunk 3]
```diff
@@ -122,21 +122,38 @@ As part of the Endpoint Detection and Response capabilities, Microsoft Defender
...
+    $ plutil -lint <PathToFile>/com.microsoft.network-extension.mobileconfig
     ```
 
+    For example, if the file was stored in the Documents:
+
+    ```bash
+    $ plutil -lint ~/Documents/com.microsoft.network-extension.mobileconfig
+    ```
+    
+    Verify that the command outputs `OK`
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md:137]:
```suggestion
    Verify that the command outputs `OK`.
```

File: windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md [hunk 4]
```diff
@@ -122,21 +122,38 @@ As part of the Endpoint Detection and Response capabilities, Microsoft Defender
...
+        
+    ```bash
+    <PathToFile>/com.microsoft.network-extension.mobileconfig: OK
+    ```
+    
 3. Follow the instructions on [this page](https://www.jamf.com/jamf-nation/articles/649/creating-a-signing-certificate-using-jamf-pro-s-built-in-certificate-authority) to create a signing certificate using JAMF’s built-in certificate authority
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies.md:143]:
```suggestion
3. Follow the instructions on [this page](https://www.jamf.com/jamf-nation/articles/649/creating-a-signing-certificate-using-jamf-pro-s-built-in-certificate-authority) to create a signing certificate using JAMF’s built-in certificate authority.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by maximvelichko** [comment]
Thank you for your contribution, Marat, it looks good.

One comment - we are updating the main documentation page (e.g. https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/mac-jamfpro-policies#step-4-configure-notifications-settings for JAMF) to have consistent instructions.) If you feel that your clarifications make sense there too - please feel free to add them there too.

**Pull request merged**